content/sse/index.textile

```diff
@@ -0,0 +1,119 @@
+---
+title: SSE and Raw HTTP Streaming API
+section: streaming-api
+index: 0
+api_separator:
+jump_to:
+  Help with:
+    - Authentication#authentication
+    - Connection state recovery#connection-state-recovery
+  API Routes:
+    - Server-sent events#sse
+    - Plain event stream#event-stream
+---
+
+The Ably SSE and raw HTTP streaming API provides a way to get a realtime stream of events from Ably in circumstances where using a full Ably Realtime client library, or even an MQTT library, is impractical.
+
+HTTP streaming allows for a request from a client to be held by a server, allowing it to push data to the client without further requests. This, much like WebSockets, help avoid the overhead involved in normal HTTP requests. "Server-sent events":https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events (SSE) is another method of achieving this for web pages, usually done through the use of "EventSource":https://developer.mozilla.org/en-US/docs/Web/API/EventSource.
+
+This feature is currently in beta. If you're interested in using this feature, or have suggestions on how we can improve it, please "get in touch":https://www.ably.io/contact.
```

Comment:
Perhaps:

This feature is currently in beta, however is ready for production loads. If you're interested in using this feature, or have suggestions on how we can improve it, we would like you to "get in touch":https://www.ably.io/contact.

Reply:
I also think this should appear in a note i.e. on the right.  I don't think it belongs in the middle of the content.